pom.xml

```diff
@@ -12,7 +12,7 @@
     <groupId>org.folio</groupId>
     <artifactId>mod-data-export-worker</artifactId>
     <description>Data Export Worker module</description>
-    <version>3.2.0-SNAPSHOT</version>
+    <version>4.0.0-SNAPSHOT</version>
```

Comment:
No need to change this manually. Snapshot version will be increased automatically by maven-release-plugin during release.